Himanshu_Sharma/project3_company/project3_company/DbModels/companyContext.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace project3_company.DbModels
+{
+    public partial class companyContext : DbContext
+    {
+        public companyContext()
+        {
+        }
+
+        public companyContext(DbContextOptions<companyContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Employee> Employee { get; set; }
+        public virtual DbSet<EmployeeProject> EmployeeProject { get; set; }
+        public virtual DbSet<Project> Project { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
```

Comment:
connection string shoud not be in db context file . It should be initialized  once in startup class and should be fetched from app.settings.json

Reply:
Fixed!